languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -26196,15 +26196,20 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
             <rule>
                 <pattern>
                     <token>auf</token>
-                    <token postag="EIG.*|UNKNOWN|PRO:REF.*|ART.*|SUB.*" postag_regexp="yes" regexp="yes">.*[a-z].*</token>
-                    <token regexp="yes">ist|war</token>
-                    <token regexp="yes" min="0">immer|stets|meist(ens)?|durchgehend</token>
+                    <token postag="EIG.*|UNKNOWN|PRO:(PER|REF).*|ART.*|SUB.*" postag_regexp="yes" regexp="yes">.*[a-z].*</token>
+                    <token inflected="yes">sein<exception>sein</exception></token>
```

Comment:
@NicoWeio, danke für die Verbesserungen!

"sein" ist auch das Lemma für das Possessivpronomen. Für dieses Token würde jetzt auch "seines" oder "seinem" gefunden werden. Lösung: Hinter `inflected="yes"` noch ein `postag="VER.*" postag_regexp="yes"` hinzufügen. Könntest du das noch machen, bitte?

Reply:
Erledigt in 8758c768363fcf6a3f428fd0e646daeafbd3b3ae.